### ISSUE TITLE
EarTrumpet crashes when Segoe fonts have Mark of Web

### ISSUE THREAD

**Issue by davuxcom** [File-New-Project/EarTrumpet#367]
A problem exists in Windows 10, where Desktop Bridge (aka Centennial) applications (like EarTrumpet) will fail to load system fonts (e.g. Segoe UI, Segoe MDL2 Assets) if they contain a Mark of the Web. This could occur, for example, after upgrading from Windows 8.1 where marked fonts were manually installed.

### [Fix a crash resulting from fonts containing a Mark of the Web](https://eartrumpet.app/help/fix-crash-resulting-from-fonts-containing-mark-of-the-web/)

**Comment by toth-Daniel** [comment]
Anything else I can do to help? Would be glad to help out any way I can :D Loved using Eartrumped when it worked and would love to start using it again :D

**Comment by riverar** [comment]
@toth-Daniel Can you try:
- Close any instances of EarTrumpet
- Download and run https://live.sysinternals.com/procmon.exe
- In the filter window, add:
  - Process Name | contains | EarTrumpet
- Click OK
- Start EarTrumpet and let it crash
- Type <kbd>CTRL</kbd><kbd>E</kbd> to stop capturing (or click the magnifying glass)
- File > Save to a PML on disk
- Upload PML to https://1drv.ms/f/s!AnjdAnZZcu-GpLFULY7DIxmMPXHc6A

**Comment by toth-Daniel** [comment]
Should be done :)

**Comment by riverar** [comment]
@toth-Daniel Can you check if `C:\Users\tothd\AppData\Local\Microsoft\Windows\Fonts\segoeui.ttf` has an unblock checkbox?

**Comment by toth-Daniel** [comment]
Ah, now THAT one does, yes :D I assume I have to unblock it? :D

**Comment by riverar** [comment]
@toth-Daniel Yeah try unblocking and start EarTrumpet.

**Comment by toth-Daniel** [comment]
Thanks dawg, now it works!!!!! Much love :)

**Comment by SvenAelterman** [comment]
Wanted to add that I just resolved the issue by unblocking SegoeUI-Light.ttf.

The manual instructions at https://eartrumpet.app/help/fix-crash-resulting-from-fonts-containing-mark-of-the-web/ don't include that file. It wasn't until I went to see which fonts had been cached in step 7 that I realized that font file was cached. It was the only font file there.

Also, step 8 (uninstall/reinstall) wasn't necessary for me.
